src/HTML_Renderer/org/lobobrowser/html/renderer/BaseElementRenderable.java

```diff
@@ -152,17 +156,65 @@ protected void invalidateLayoutLocal() {
   private int lastAvailWidthForDeclared = -1;
   private int lastAvailHeightForDeclared = -1;
 
-  protected Integer getDeclaredWidth(final RenderState renderState, final int actualAvailWidth) {
+  protected Integer getDeclaredWidth(final RenderState renderState,
+      final int actualAvailWidth) {
     Integer dw = this.declaredWidth;
-    if ((dw == INVALID_SIZE) || (actualAvailWidth != this.lastAvailWidthForDeclared)) {
+    if ((dw == INVALID_SIZE)
+        || (actualAvailWidth != this.lastAvailWidthForDeclared)) {
       this.lastAvailWidthForDeclared = actualAvailWidth;
-      final int dwInt = this.getDeclaredWidthImpl(renderState, actualAvailWidth);
+      final int dwInt = this.getDeclaredWidthImpl(renderState,
+          actualAvailWidth);
       dw = dwInt == -1 ? null : new Integer(dwInt);
       this.declaredWidth = dw;
     }
     return dw;
   }
 
+  protected Integer getDeclaredHelper(final RenderState renderState,
```

Comment:
This and the following four functions (getDeclared*) seem to have been newly added but not used. Looks like a copy-paste error. Can you remove them and verify there are no compilation errors?